specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/examples/PolicyEvents_QueryManagementGroupScope.json

```diff
@@ -8,6 +8,7 @@
   "responses": {
     "200": {
       "body": {
+        "@odata.nextLink": null,
         "@odata.context": "https://management.azure.com/providers/Microsoft.Management/managementGroups/myManagementGroup/providers/Microsoft.PolicyInsights/policyEvents/$metadata#default",
         "@odata.count": 2,
         "value": [
```

Comment:
>value [](start = 9, length = 5)

Please also add "complianceState" field to the response. Same in all events json for this version.